src/App.js

```diff
@@ -9,25 +9,28 @@ import {
 } from 'react-router-dom';
```

Comment:
Line 3 (import Firebase Test) can be removed since it isn't being used.

Reply:
done!